README.md

```diff
@@ -1,13 +1,23 @@
 # Trello KPI Dashboard
-
 A simple project to collect metrics from a [Trello](https://www.trello.com) board and create a web-based KPI dashboard
+Installation
+----
+Clone the [repository](https://github.com/inventivegroup/interns-oct2018)
 
+With web-services as working directory, install npm dependencies with this command:
+```
+npm install
+```
 npm Commands
 ----
 To sync the local client directory with the s3 bucket, use this command:
```

Comment:
We should have different package.json files. One for the client which knows how to push the client code to S3 and one for the serverless/backend tasks.